lib/xmlSchemaFaker.js

```diff
@@ -46,17 +55,19 @@ function convertSchemaToXML(name, schema, attribute, indentChar, inden) {
     }
     retVal += '>';
     retVal += childNodes;
-    retVal += '\n</' + name + '>';
+    retVal += `\n${cInden}</${tagPrefix+name}>`;
   }
   else if (schema.type === 'array') {
     // schema.items must be an object
     var isWrapped = _.get(schema, 'xml.wrapped'),
       extraIndent = isWrapped ? 1 : 0;
     var arrayElemName = _.get(schema, 'items.xml.name', name, 'arrayItem');
-    var contents = convertSchemaToXML(arrayElemName, schema.items, false, indentChar, inden + extraIndent) + 
-      convertSchemaToXML(arrayElemName, schema.items, false, indentChar, inden + extraIndent);
+    var schemaItemsWithXmlProps = _.cloneDeep(schema.items);
```

Comment:
Let's combine all var definitions into one. i.e. we're each time using `var x = y;` instead of

```
var x = a,
  y = b,
  ...
  z = c;
```